src/expressions.rs

```diff
@@ -1,18 +1,16 @@
 #![allow(clippy::unused_unit)]
-use polars::prelude::*;
```

Comment:
Should the polars imports not be removed? They are used for the string functions e.g. `single_space`

Reply:
Ah yeah, I might have removed before the merge, thanks

Reply:
Fixed in latest commit.